src/main/java/Parser.java

```diff
@@ -123,6 +127,10 @@ public void deleteTask(String command) {
         String[] commandlist = command.split(" ");
         /* check for exceptions */
         Integer taskNum = Integer.valueOf(commandlist[1]);
+
+        assert taskNum < this.model.getTotalTasks() 
```

Comment:
hmm something looks off here :///
hmmmmmmmm yous reu this is right? i dun wanna break anything